app/assets/v2/js/pages/dashboard.js

```diff
@@ -228,27 +233,27 @@ var process_stats = function(results) {
 
   switch (num) {
     case 0:
-      $('#matches').html(gettext('No Results'));
+      // $('#matches').html(gettext('No Results'));
       $('#funding-info').html('');
       break;
     case 1:
-      $('#matches').html(num + gettext(' Matching Result'));
+      // $('#matches').html(num + gettext(' Matching Result'));
       $('#funding-info').html("<span id='modifiers'>Funded Issue</span><span id='stats' class='font-body'>(" + stats + ')</span>');
       break;
     default:
-      $('#matches').html(num + gettext(' Matching Results'));
+      // $('#matches').html(num + gettext(' Matching Results'));
       $('#funding-info').html("<span id='modifiers'>Funded Issues</span><span id='stats' class='font-body'>(" + stats + ')</span>');
   }
 };
 
 var paint_bounties_in_viewport = function(start, max) {
```

Comment:
we can probalby get rid of the whole `paint_bounties_in_viewport()` abstraction given that we're only doing 20 results at a time now. 

1. its too much complexity. 
1. it also breaks the infinite scroll because there might be bounties in the buffer that need to be painted when the user hits the bottom of the screen (as oppposed to if there were no buffer, where you'd know we need to fetch more bounties if the user is at the bottom of the screen)

Reply:
on second thought, looks like you may have handled this adequately below

Reply:
I left the paint_bounties_in_viewport funciton in so you can tweak the ratio of api calls to rendering. The avatars get loaded up when you view them, so you might want to call the api every 40 times items, and only have the avatars load every 10 items so you don't have a bunch of http calls at once.